### DISCUSSION TITLE
So how do i make this work exactly?

### DISCUSSION THREAD

**Discussion by RianaStarspaw** [oobabooga/text-generation-webui#2972] [General]
Hi, ive tried to follow couple tutorials from youtube to adding models into this webui but i just keep getting random python errors and nothing working...

**Comment by viperwasp** [comment]
I may not be able to reply to often. But I can attempt to slowly help. First did you install via the one click installer on found on this page? 
https://github.com/oobabooga/text-generation-webui

Did you run the start_windows.bat file? Do you have an NVIDIA GPU? And did you choose Nvidia. If you don't have an NVIDIA GPU please tell us what you have. 

You should also run update_windows.bat to update to most recent version after you install oobabooga. 
Also copy and past the error or at least the last 5-10 lines of the CMD LOG/ERROR you get.

**Comment by luthis1124** [comment]
What operating system are you using? Can you post some errors? Do you get to the web ui etc?

**Comment by RianaStarspaw** [comment]
> I may not be able to reply to often. But I can attempt to slowly help. First did you install via the one click installer on found on this page? https://github.com/oobabooga/text-generation-webui
> 
> Did you run the start_windows.bat file? Do you have an NVIDIA GPU? And did you choose Nvidia. If you don't have an NVIDIA GPU please tell us what you have.
> 
> You should also run update_windows.bat to update to most recent version after you install oobabooga. Also copy and past the error or at least the last 5-10 lines of the CMD LOG/ERROR you get.

Hello, yes i ran the start_windows.bat file, i have an NVIDIA GPU (RTX 3060) and did choose that.
Didnt try touch the update_windows.bat 



> What operating system are you using? Can you post some errors? Do you get to the web ui etc?
Windows 10, ive got to the webui until i loaded this model: https://huggingface.co/TheBloke/Wizard-Vicuna-30B-Uncensored-GPTQ

Unfortunately at the moment i wont have time to post any error log but will do it later after work, i know i should have started with this but only thought about write here after i deleted the entire webui folder o_O

**Reply by missionfloyd** [reply]
> i have an NVIDIA GPU (RTX 3060)
> ive got to the webui until i loaded this model: https://huggingface.co/TheBloke/Wizard-Vicuna-30B-Uncensored-GPTQ

Sounds like #632. Have you tried smaller models ([7B](https://huggingface.co/TheBloke/Wizard-Vicuna-7B-Uncensored-GPTQ) / [13B](https://huggingface.co/TheBloke/Wizard-Vicuna-13B-Uncensored-GPTQ))?
